packages/pipeline/src/Pipeline.tsx

```diff
@@ -0,0 +1,285 @@
+import React, {
+  forwardRef,
+  createRef,
+  useState,
+  Ref,
+  ReactElement,
+  ReactNode,
+} from 'react';
+
+import PropTypes from 'prop-types';
+import { findLast } from 'lodash';
+
+import { uiColors } from '@leafygreen-ui/palette';
+import Tooltip from '@leafygreen-ui/tooltip';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { useElementNode, useMutationObserver } from '@leafygreen-ui/hooks';
+
+import Stage from './Stage';
+import Counter from './Counter';
+
+import {
+  getPipelineCounterTooltip,
+  isStageElement,
+  isElementOverflowed,
+} from './utils';
+
+import {
+  getRootStyle,
+  Size,
+  Variant,
+  layout,
+  colors,
+  pipelineAttr,
+  pipelineStages,
+  stageChevronAttr,
+} from './styles';
+
+export interface StateForStyles {
+  hasHiddenStages: boolean;
+  size: Size;
+  variant: Variant;
+}
+
+export interface PipelineProps {
```

Comment:
Don't need to export this either. Nothing is directly using the interface, it's just being used by the component. 

If we had another component that wanted to extend the interface or directly access it then we'd export the interface, however we don't need that right now.

Reply:
Resolved by 98f6cdd